### PR TITLE
Beautify the project page

### PR DESCRIPTION
**Pull request by GrantPerkins**
Individual components still need to be fixed.

### CODE REVIEW

File: packages/client/src/components/dataset/Dataset.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import MoreVertIcon from "@material-ui/icons/MoreVert";
 import { LazyLoadImage, ScrollPosition, trackWindowScroll } from "react-lazy-load-image-component";
 import { GetDataset, GetDataset_dataset_images, GetDatasetVariables } from "./__generated__/GetDataset";
 import { useQuery } from "@apollo/client";
-import RenameDatasetDialogButton from "./RenameDatasetDialog";
+import RenameDatasetDialogButton from "./RenameProjectDialog";
```
Comment (NLZ432) [inline packages/client/src/components/dataset/Dataset.tsx]: Was importing dataset buttons from a project file intentional?

Comment (NLZ432) [reply packages/client/src/components/dataset/Dataset.tsx]: Has this been a thing for a while and I havn't noticed?

File: packages/client/src/components/project/RenameDatasetDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+const RENAME_PROJECT_MUTATION = gql`
+  mutation RenameProject($id: ID!, $newName: String!) {
+    renameProject(id: $id, newName: $newName) {
+      id
+      name
+    }
+  }
+`;
+
+export default function RenameProjectDialog(props: { id: string }): ReactElement {
```
Comment (NLZ432) [inline packages/client/src/components/project/RenameDatasetDialog.tsx]: Could you rename this file to RenameProjectDialog.tsx?

**Review by NLZ432 — CHANGES_REQUESTED** [review]

File: packages/client/src/components/dataset/Dataset.tsx (hunk 1 above)
Comment (GrantPerkins) [reply packages/client/src/components/dataset/Dataset.tsx]: filename was wrong

File: packages/client/src/components/project/RenameDatasetDialog.tsx (hunk 2 above)
Comment (GrantPerkins) [reply packages/client/src/components/project/RenameDatasetDialog.tsx]: yes

**Review by NLZ432 — APPROVED** [review]
noice

**Pull request merged**